src/main/java/seedu/address/github/GithubApi.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.github;
```

Comment:
do you think two packages with the same name could potentially be confusing?

Reply:
I think this package name is fine. But might be better if we put inside `logic`